### PR TITLE
[v0.16] Hash for pipe::Meta

### PR DESCRIPTION
**Pull request by kvark**
Allows hashing and comparing pipeline states.

### CODE REVIEW

File: src/render/Cargo.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 The Gfx-rs Developers.
+# Copyright 2017 The Gfx-rs Developers.
```
Comment (torkleyy) [inline src/render/Cargo.toml]: You sure? :)

Comment (kvark) [reply src/render/Cargo.toml]: you got me! Need to adjust that time machine a bit, having off by 2 errors :P

**Comment by kvark** [comment]
bors r=torkleyy

**Comment by kvark** [comment]
Published in https://crates.io/crates/gfx/0.16.3

**Pull request merged**